contributing.md

```
@@ -0,0 +1,38 @@
+pip install pre-commit
+
+# 3) Install the git hook
+pre-commit install
+
+# 4) (First time) Run on the whole repo and commit any fixes
+pre-commit run --all-files
+git add -A && git commit -m "chore: apply pre-commit fixes"
+```
+
+## Running Specific Checks
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)